packages/shorebird_cli/lib/src/commands/patch/patch_command.dart

```diff
@@ -77,8 +78,8 @@ class PatchCommand extends ShorebirdCommand {
       ..addOption(
         'release-version',
         help: '''
-The version of the associated release (e.g. "1.0.0"). This should be the version
-of the iOS app that is using this module.''',
```

Comment:
I don't think we want to completely remove this text, although how it was is confusing. Maybe:

```
The version of the associated release (e.g. "1.0.0").
To target the latest release (highest released version) use --release-version=latest.

If you are building an xcframework or aar, this number needs to match the host app's release version.
```

Reply:
thanks for the context, done 👍